File: pickit/src/main/java/com/hbisoft/pickit/CallBackTask.java
package com.hbisoft.pickit;

interface CallBackTask {
    void PickiTonUriReturned();
    void PickiTonPreExecute();
    void PickiTonProgressUpdate(int progress);
    void PickiTonPostExecute(String path, boolean wasDriveFile, boolean wasSuccessful, String reason);
}

File: pickit/src/main/java/com/hbisoft/pickit/CallBackTask.kt
package com.hbisoft.pickit

internal interface CallBackTask {
    open fun PickiTonUriReturned()
    open fun PickiTonPreExecute()
    open fun PickiTonProgressUpdate(progress: Int)
    open fun PickiTonPostExecute(path: String?, wasDriveFile: Boolean, wasSuccessful: Boolean, reason: String?)
}